docs/references/strategies/languages/clojure/clojure.md

```diff
@@ -3,9 +3,9 @@
 When developing in Clojure, [leiningen](https://leiningen.org/) is the most common package manager. Dependencies are specified in a manifest file by users which is used by the `lein` tool to build a dependency graph and download the correct dependencies.
 
 
-| Strategy    | Direct Deps        | Transitive Deps          | Edges              | Container Scanning (Experimental) |
```

Comment:
I didn't even realize you added this column for "Container Scanning", this is awesome.